server/src/internalClusterTest/java/org/elasticsearch/action/bulk/BulkIntegrationIT.java

```diff
@@ -218,14 +217,14 @@ public void testDeleteIndexWhileIndexing() throws Exception {
         }
     }
 
-    public void testMixedAutoCreate() {
+    public void testMixedAutoCreate() throws Exception {
         Settings settings = Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0).build();
 
         PutIndexTemplateV2Action.Request createTemplateRequest = new PutIndexTemplateV2Action.Request("logs-foo");
         createTemplateRequest.indexTemplate(
             new IndexTemplateV2(
                 List.of("logs-foo*"),
-                new Template(settings, null, null),
+                null,
```

Comment:
I think this makes `settings` unused? But I am also uncertain of why this change was made? Just cleanup or a necessity. If the latter, I would like to understand why.

Reply:
This was just a cleanup. I will remove the settings variable.

(copy paste from `testAutoCreateV1TemplateNoDataStream()` which ensures that a template applied, but that is not needed here, because if the data stream is created then the template has been applied)